README.md

```diff
@@ -92,6 +92,8 @@ $ docker run -d \
 nextcloud
 ```
 
+Please note that anything inside `/var/www/html` will be removed by the installation script, except for `config`, `data`, `custom_apps`, `themes` and `version.php`. So make sure you do not leave other important files inside.
```

Comment:
Can you just refer to [upgrade.exclude](https://github.com/nextcloud/docker/blob/master/upgrade.exclude)?

Reply:
Yeah, I agree. That would probably be better.

Reply:
```suggestion
Please note that anything inside `/var/www/html` will be removed by the installation script, except for those listed in [upgrade.exclude](https://github.com/nextcloud/docker/blob/master/upgrade.exclude). So make sure you do not leave other important files inside.
```